packages/mdctl-axon-tools/lib/StudyManifestTools.js

```diff
@@ -707,7 +707,7 @@ class StudyManifestTools {
         }
         case 'ec__document_template': {
           // Get the eConsents ID's from the study or the manifest
-          ids = (await this.getObjectIDsArray(org, key, property, values)).map(v => v._id)
+          ids = (await this.getObjectIDsArray(org, key, 'ec__study', values)).map(v => v._id)
```

Comment:
Hi @emilyackermanmedable,I think that _property_ variable is used to determine if querying by 'c_study' (for full migrations) or 'c_key' (for partial migrations) when calling getObjectIDsArray method. 
If that value is hard coded , will it still work properly for partial migrations where we pass c_key instances instead?

Reply:
Fixed it - thank you for catching that!